app/php/composer.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from app.apache.apache import Apache
+from app.apache.proxyPass import ProxyPass
+
+
+class Composer:
+
+    def run(self, serverName=None, port=None):
```

Comment:
Divide run into 4 functions. All system calls alone as some function. Call it setup(). 
And then as mentioned below make a function called seed().
And then a function called start()
And then a function called restart(), which will stop and start the server again.